app/src/main/java/pl/ahyoshin/demo/UserRepository.java

```diff
@@ -15,8 +16,10 @@ public UserRepository(EntityManager em) {
     }
 
     @Transactional
-    public void saveUser(){
+    public void saveUser(RegisterRequest registerRequest){
         var userEntity = new UserEntity();
+        userEntity.setUsername(registerRequest.getUsername());
+        userEntity.setPassword(registerRequest.getPassword());
         em.persist(userEntity);
```

Comment:
Wolimy tak jak jest czy tak?
```suggestion
    public void saveUser(UserEntity userEntity){
        em.persist(userEntity);
```